os/board/esp_wrover_kit/Kconfig

```diff
@@ -0,0 +1,204 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+if ARCH_BOARD_ESP_WROVER_KIT
+
+choice
+	prompt "On-board Crystal Frequency"
+	default ESP32CORE_XTAL_40MZ
+
+config ESP32CORE_XTAL_40MZ
+	bool "40MHz"
+
+config ESP32CORE_XTAL_26MHz
```

Comment:
1. are there two kinds of ESP WROVER KIT? One has 40MHz and another has 26MHz?
    Or does it determine working freq.? Is there no way to change it on run-time? Is it static?
2. 40MZ -> 40MHz

Reply:
First,  i will modify the configuration name. 
Secordly, for XTAL and CPU frequency, so far it's set in bootloader and not be implemented in ESP32 chip code.  Now this branch is only for bringing up esp32 board. We will add cpu frequency settings later.
Thanks.